util/src/main/scala/com/scalableminds/util/mvc/ExtendedController.scala

```diff
@@ -177,6 +177,7 @@ trait MimeTypes {
   val xmlMimeType: String = "application/xml"
   val zipMimeType: String = "application/zip"
   val jsonMimeType: String = "application/json"
+  val formUrlEncoded: String = "application/x-www-form-urlencoded"
```

Comment:
```suggestion
  val formUrlEncodedMimeType: String = "application/x-www-form-urlencoded"
```
(for consistency with the others)